### ISSUE TITLE
User interface improvements

### ISSUE THREAD

**Issue by nkaretnikov** [microsoftfeedback/WinDbg-Feedback#56]
Hello,

I'd like to propose the following UI improvements:
- Reduce the size of some icons (see Local Help and Timelines, for instance, but there's more on other panes).
- Take advantage of available space on the right (see the screenshot below).
- Reduce the font size in some windows (see Timelines, Memory; see the screenshot below). It doesn't respect user settings at all.
- Why is the button next to "Unable to access pseudo-register" not aligned with the text?
- Toggle Initial Break is not saved between Restarts.
- Switch back to the old WinDbg icon. It's much better, just make it look good on modern screens. The same for other icons. Some of them are very blurry or hard to parse visually.
- Allow to create custom themes. I like the Dark one, but it's still too contrast for me. But I want to do much more. I want to have something like Vim Solarized Dark.
- There's a bug with Source debugging. If you stick the Source window above the Command one (not on top), another Source window still pops up on the side (vertical split) when you hit a breakpoint.
- Allow to configure shortcuts for everything and display them in the UI (where possible) to make them easier to learn.
  For instance, Ctrl-Break to abort a command. It's not possible to use this on some keyboards. I also want Unix-style shortcuts for text navigation, aborting commands, searching history, etc.
- Provide quick help for all commands in the Command window directly (not just via `.hh`).

I really like the old UI better. There's a lot of wasted space in the new one. The only reason I use Preview is just to play around with time-travel debugging and the debugger data model.

Version: WinDbg 1.0.2007.06001 installed from the Microsoft Store.

Thanks!

<img width="676" alt="windbg_ui2" src="https://user-images.githubusercontent.com/10469885/94106654-7b6a0b80-fe44-11ea-8acd-1625ac02b593.png">
<img width="1104" alt="windbg_ui" src="https://user-images.githubusercontent.com/10469885/94106658-7efd9280-fe44-11ea-9701-edc2c163a413.png">

**Comment by nkaretnikov** [comment]
Also, there is some issue with the Timelines window. Why am I seeing this spinning circle, which never goes away?

<img width="659" alt="windbg_timelines" src="https://user-images.githubusercontent.com/10469885/94373949-80afaa80-0111-11eb-842e-076e921d68f7.png">

**Comment by jsschultz** [comment]
Hi @nkaretnikov , it has been years since this feedback, but I just wanted to close the loop. Thank you for your feedback! We have fixed some of the reported issues and others are still on our backlog.

To highlight some of the fixes mentioned:

The ribbon has a new "Simplified" mode in addition to "Classic" mode, which can be toggled via the ribbon's context menu.

<img width="480" height="143" alt="Image" src="https://github.com/user-attachments/assets/d1222862-7587-4009-97d7-5ccc616ccf89" />


The command window has auto suggestions when typing commands (starting with !, ., or $)

<img width="230" height="238" alt="Image" src="https://github.com/user-attachments/assets/4d04d811-8898-472c-90dc-424c01e11f18" />
